docs/diff/rules.md

```diff
@@ -0,0 +1,190 @@
+      "apiContract:binding": "query",
+      "raml-shapes:schema": {
+        "@id": "#/web-api/end-points/%2Forganizations%2F%7BorganizationId%7D%2Fproducts/get/request/parameter/newParam/scalar/schema"
+      }
+    },
+    "removed": {},
+    "rule": {
+      "name": "Rule to detect required parameter addition",
+      "type": "required-parameter-added",
+      "params": {
+        "category": "Breaking"
+      }
+    }
+  }
+ ]
+```
+
+Rules to categorize the above differences. Each node difference is a fact which is called as “diff” in the rule.
+
+```
+[
+  {
+    "name": "Rule to detect version change",
+    "conditions": {
+      "all": [
+        {
+          "fact": "diff",
+          "path": "$.type",
+          "operator": "contains",
+          "value": "apiContract:WebAPI"
+        },
+        {
+          "fact": "diff",
+          "path": "$.added",
+          "operator": "hasProperty",
+          "value": "core:version"
+        },
+        {
+          "fact": "diff",
+          "path": "$.removed",
+          "operator": "hasProperty",
+          "value": "core:version"
+        }
+      ]
+    },
+    "event": {
+      "type": "version-changed",
+      "params": {
+        "category": "Breaking"
+      }
+    }
+  },
+  {
+    "name": "Rule to detect required parameter addition",
+    "conditions": {
+      "all": [
+        {
+          "fact": "diff",
+          "path": "$.type",
+          "operator": "contains",
+          "value": "apiContract:Parameter"
+        },
+        {
+          "fact": "diff",
+          "path": "$.added",
+          "operator": "hasProperty",
+          "value": "core:name"
+        },
+        {
+          "fact": "diff",
+          "path": "$.added.apiContract:required",
+          "operator": "equal",
+          "value": true
+        },
+        {
+          "fact": "diff",
+          "path": "$.removed",
+          "operator": "hasNoProperty",
+          "value": "core:name"
+        }
+      ]
+    },
+    "event": {
+      "type": "required-parameter-added",
+      "params": {
+        "category": "Breaking"
+      }
+    }
+  },
+  {
+    "name": "Rule to ignore changes in examples",
+    "conditions": {
+      "any": [
+        {
+          "fact": "diff",
+          "path": "$.type",
+          "operator": "contains",
+          "value": "apiContract:Example"
+        }
+      ]
+    },
+    "event": {
+      "type": "example-changed",
+      "params": {
+        "category": "Ignored"
+      }
+    }
+  }  
+]
+```
+
+ [json-rules-engine](https://www.npmjs.com/package/json-rules-engine) is used to apply the rules. Read the [rules document](https://github.com/CacheControl/json-rules-engine/blob/master/docs/rules.md) to understand more about the rules structure
+
+### Custom Operators
+
+json-rules-engine allows defining of custom operators to use along with the default ones. Diff tool library has the following custom operators that you can use in your rules.
```

Comment:
Suggest changing from 'json-rules-engine allows defining...' to The 'json-rules-engine lets you define either custom or default operators.'
Suggest changing from 'Diff tool library has the ..." to 'The Diff tool library offers these customer offers  that you can use in your rules.'